tutorials/prometheus-adapter.md

```diff
@@ -309,39 +333,22 @@ docker exec -it pg_prometheus bash
 Once in the container use psql `psql postgres postgres` to connect to
 TimescaleDB and run SQL queries. Below you can find some example queries.
 
-**CPU frequency values for core “0” for the last 5 minutes (gauge):**
-
-PromQL:
-```
-node_cpu_frequency_hertz{cpu="0"}[5m]
-```
-
-SQL:
-```sql
-SELECT time, value
-FROM metrics
-WHERE name='node_cpu_frequency_hertz' AND labels->>'cpu' = '0' AND
-      time > NOW() - interval '5 min'
-```
-
-
-**Max CPU frequency by CPU core in the last 5 minutes:**
+**Get total bytes received by the eth0 network interface (resets to 0 if machine restarts):**
 
 PromQL:
 ```
-max(max_over_time(node_cpu_frequency_hertz[5m])) by (cpu)
+node_network_transmit_bytes_total{device="eth0"}
 ```
 
 SQL:
 ```sql
-SELECT labels->>'cpu', MAX(value)
-FROM metrics
-WHERE name='node_cpu_frequency_hertz' AND
-    time > NOW() - interval '5 min'
-GROUP BY labels->>'cpu';
+SELECT time, value AS "total transmitted bytes" 
```

Comment:
hmm, was this query previously incorrect?

Reply:
Yes; the node_cpu_frequency_hertz metric isn't exported in the Docker container.